sbt-dotty/sbt-test/sbt-dotty/example-project/test

```diff
@@ -1,4 +1,4 @@
 > run
 > 'set initialCommands := "1 + 1" '
 # FIXME: does not work on the CI
-#> console
```

Comment:
The test is only run on nightly. You need to uncomment:
https://github.com/lampepfl/dotty/blob/0d7826c8a77bfa0da727ac21213475739faa9428/.drone.yml#L45-L47

Reply:
Thanks.